### DISCUSSION TITLE
Matek H743

### DISCUSSION THREAD

**Discussion by Grumpyrse** [iNavFlight/inav#7004] [Q&A]
Will Matek H743 be supported at all like it was promised with 3.0 or has this been shelved??

**Comment by JulioCesarMatias** [comment]
> Will Matek H743 be supported at all like it was promised with 3.0 or has this been shelved??

H7 is available in the source repository; target Hex files will be released as targets mature, probably in the future point releases.

**Reply by ThisIsMyNameBecauseEverythingIsTaken** [reply]
What is missing from making the H743-WING a fully supported target? How stable is the current code? Any major features missing?

**Reply by JulioCesarMatias** [reply]
https://github.com/iNavFlight/inav/issues/6371

**Reply by ThisIsMyNameBecauseEverythingIsTaken** [reply]
Thank you. Once my board arrives in the mail I'll see if I can help with any of those.
